testsuite/features/secondary/trad_migrate_to_sshminion.feature

```diff
@@ -2,6 +2,7 @@
 # Licensed under the terms of the MIT license.
 
 @scope_traditional_client
+@scope_sp_migration
```

Comment:
false -- this is NOT about SP migration

this is about migration from trad to minion and back

yeah, I know, we deal with many different kinds of "migrations" :-)

Reply:
ah, right